test/example/logger/demo_log_text_error.f90

```diff
@@ -0,0 +1,24 @@
+program demo_log_text_error
+    use stdlib_logger
+
+    character(*), parameter :: filename = 'dummy.txt'
+    integer                 :: col_no, line_no, lun
+    character(128)          :: line
+    character(*), parameter :: message = 'Bad text found.'
+
+    open (newunit=lun, file=filename, status='old', &
+          form='formatted')
+    line_no = 0
+    do
+        read (lun, fmt='(a)', end=900) line
+        line_no = line_no + 1
+        call check_line(line, status, col_no)
```

Comment:
This function `check_line` doesn't exist in the program scope neither the `status` variable